examples/deployment/algob.config.js

```diff
@@ -67,11 +77,14 @@ let defaultCfg = {
   port: 46469,
   token: "aade468d25a7aa48fec8082d6a847c48492066a2741f3731e613fdde086cd6e9",
   accounts: accounts,
+  kmdConfig: kmdconfig,
+  kmdAddresses: await loadKMDAccounts(kmdconfig.host, kmdconfig.token, kmdconfig.port, 
+    kmdconfig.walletname, kmdconfig.walletpassword)
```

Comment:
can we pass it `loadKMDAccounts(kmdconfig)` like this ?

Reply:
No, this is wrong. It shouldn't be here. I'm removing this.